packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -215,9 +215,9 @@ class ScrollPosition extends ViewportOffset {
   /// scroll notifications will be dispatched. No overscroll notifications can
   /// be generated by this method.
   ///
-  /// Immediately after the jump, a ballistic activity is started, in case the
-  /// value was out of range.
-  void jumpTo(double value) {
+  /// If settle is true then, immediately after the jump, a ballistic activity
+  /// is started, in case the value was out of range.
+  void jumpTo(double value, { bool settle: true }) {
```

Comment:
This feature is scary. It's unstable. Many, many things in the ScrollPosition logic automatically start ballistic activities.

I don't think we should have a way to explicitly enter this kind of non-stable state. It will lead to very brittle UIs.

Can you elaborate on the use case for this? Maybe there's a better solution.